test/Test/Pos/Ssc/GodTossing/VssCertDataSpec.hs

```diff
@@ -34,6 +40,29 @@ spec = describe "Ssc.GodTossing.VssCertData" $ do
         prop description_verifySetLastKnownSlot verifySetLastKnownSlot
     describe "verifyDeleteAndFilter" $
         prop description_verifyDeleteAndFilter verifyDeleteAndFilter
+    let smaller = modifyMaxSuccess (const 100)
+    describe "verifyRollback" $ smaller $
+        prop description_verifyRollback $
+            do goodVssCertData@(VssCertData {..}) <- getVssCertData <$> arbitrary
```

Comment:
I think it's better to move this code into separate function as it's done usually.